### ISSUE TITLE
Cannot get yasnippet to work

### ISSUE THREAD

**Issue by fluca1978** [syl20bnr/spacemacs#14154]
#### Description :octocat:
I cannot load any snippet from yasnippet.

#### Reproduction guide :beetle:
- Start Emacs
- edit a file
- `M-m i s` to insert a snippet

*Observed behaviour:* :eyes: :broken_heart:
The popup menu of the snippets is empty and does not provide any snippet to be completed with.
I've run several times `yas-reload-all` and restarted Emacs but no completion snippets are there.
Snippets are within my `private` directory

```
% ls -R ~/.emacs.d/private/snippets 
/home/luca/.emacs.d/private/snippets:
fluca1978.txt  markdown  README.md

/home/luca/.emacs.d/private/snippets/markdown:
pwc.snippet.md
```

and I've set the searching directory as follows in my `.spacemacs`:

```
(setq yas-snippet-dirs
      '("~/.emacs.d/private/snippets"          ;; personal snippets
        "~/.emacs.d/private/snippets/markdown"
        ))
```

in the desperate effort to force yasnippet to locate my scripts.
However, for some reason, it does ignore my settings since when I execute `yas-reload-all` I got this in the messages buffer

```
[yas] Check your `yas-snippet-dirs': /home/luca/.emacs.d/elpa/yasnippet-20200604.246/snippets is not a directory
[yas] Snippets loaded with some errors.  Check *Messages*.
```

*Expected behaviour:* :heart: :smile:
`M-m s i` should provide me helm completion about snippets on my directory tree.

#### System Info :computer:
- OS: gnu/linux
- Emacs: 26.3
- Spacemacs: 0.200.13
- Spacemacs branch: master (rev. d46eacd83)
- Graphic display: t
- Distribution: spacemacs
- Editing style: emacs
- Completion: helm
- Layers:
```elisp
((colors :variables)
 sql python helm auto-completion emacs-lisp git markdown org
 (shell :variables shell-default-height 40 shell-default-position 'bottom)
 spell-checking syntax-checking version-control)
```
- System configuration features: XPM JPEG TIFF GIF PNG SOUND DBUS GSETTINGS GLIB NOTIFY LIBSELINUX GNUTLS FREETYPE XFT ZLIB TOOLKIT_SCROLL_BARS GTK3 X11 XDBE XIM THREADS

**Comment by fluca1978** [comment]
Manually creating the directory from the terminal made yasnippet happy to load the snippets:

```
% mkdir -p /home/luca/.emacs.d/elpa/yasnippet-20200604.246/snippets
```

however, I still don't have any snippet in the completion, and after a `yas-reload-all` I have `[yas] Snippets loaded successfully.` but still nothing working on `M-m i s` (in the powerline I see `(1/0) 0 total`.

**Comment by fluca1978** [comment]
I've noted, and I think it is somehow related, is that if I execute `helm-yas-create-snippet-on-region` I got:

```
helm-M-x-execute-command: Wrong type argument: stringp, yas-installed-snippets-dir
```

I reviewed my configuration and found that the `yasnippet-snippets` package was loaded into the customize section instead of the `dotspacemacs-additional-packages`.
Adding it to it and reloading emacs has fixed the problem and installed all the snippets.
Sorry about the noise.
